scripts/post_leap.sh

```diff
@@ -35,4 +35,11 @@ if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/deploy-rpc.complete" ]]; then
 else
   log "deploy-rpc" "skipped"
 fi
+
+if [ "$UPGRADE_ELASTICSEARCH"="yes" ]; then
```

Comment:
if [ "$UPGRADE_ELASTICSEARCH" == "YES" ];

Reply:
get it, will fix the equals. But 'yes' is made on purpose to pair with upstream format: https://github.com/rcbops/rpc-openstack/blob/newton/scripts/functions.sh#L62 all vars are exported in lower case.

Reply:
yeah, that's fine, mainly was mentioning the double =